README.md

```diff
@@ -310,7 +310,6 @@ layout or page's [front matter] YAML in the following form:
 
 ```yml
 scripts:
-  - /path/to/script.js
```

Comment:
I think the intent here is that you can use either syntax, just the path, or the more detailed approach where you specify the `src` and `async` option.